packages/button/src/ButtonContent/ButtonContent.styles.ts

```diff
@@ -70,3 +70,37 @@ export const buttonContentSizeStyle: Record<Size, string> = {
     gap: 8px;
   `,
 };
+
+export const textlessLoadingStyles = (
+  loadingContentRef: React.MutableRefObject<HTMLDivElement | null>,
+) => css`
+  width: ${loadingContentRef.current?.style.width};
+  position: relative;
+`;
+
+export const centeredSpinnerStyles = css`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+export const buttonSpinnerSize: Record<Size, number> = {
+  [Size.XSmall]: 16,
+  [Size.Small]: 16,
+  [Size.Default]: 16,
+  [Size.Large]: 20,
+};
+
+export const spinnerStyles: Record<Theme, string> = {
+  [Theme.Dark]: css`
+    path {
```

Comment:
should we be more specific than this?

Reply:
or provide a better way in the spinner API to change the color?

Reply:
or provide a better way in the spinner API to change the color?

Reply:
`colorOverride`

Reply:
Fine by me either way. Spinner only has one path which makes the selector simple, so it feels like a non-issue. I think it's only going to be overridden internally too. I'm happy to add it if others agree with that approach

Reply:
i like the `colorOverride` approach!